### ISSUE TITLE
Activated filter categories are not perceivable by screen readers

### ISSUE THREAD

**Issue by anirudhabhowmikhpe** [grommet/grommet#7875]
Whatever filter is activated, it should announce by screen reader when focus will be on filter button.

**Description:** 
There are different types of filters on the platform. On the "Devices" page, buttons must be activated to display only content from the activated category in a table displayed below. Screen readers cannot detect which buttons are activated and which are not.

**Example:** Active filters are not perceivable in device subscriptions.

**Fix:** The status of the buttons should be perceivable by screen readers. For example, the class "sr-only" or an aria-label can be used to make the status perceptible only to screen readers. Example: <span class="sr-only"> Selected </span>.
Alternatively, the filters should be implemented using interactive standard HTML elements, e.g.,
with a drop-down list <select>.

**Please take reference of this ticket:**
https://hpe.atlassian.net/browse/GLCP-297230

![Image](https://github.com/user-attachments/assets/d1365c17-e8a5-42d2-954b-89a4f9d38613)

**Comment by halocline** [comment]
Hi @anirudhabhowmikhpe - Thank you for submitting this issues and for being committed to creating accessible experiences.

This needs to be addressed in the implementation instance. You may find this [quick filtering + selector pattern](https://design-system.hpe.design/templates/selector#quick-filtering) and [code sample](https://github.com/grommet/hpe-design-system/blob/master/apps/docs/src/examples/templates/selector/QuickFilter.js) as a useful reference.

**Comment by jcfilben** [comment]
Marking as closed, see above comment.
